contracts/pallet-ibc/src/benchmarks/tendermint_benchmark_utils.rs

```diff
@@ -462,6 +462,7 @@ where
 			)
 			.unwrap(),
 			delay_period,
+			host_consensus_state_proof: vec![],
```

Comment:
Do these benchmarks don’t really matter? Otherwise we should be able to add a real proof

Reply:
We can't add a real proof at the moment, that would require modifying the blockhash storage in frame system somehow, which we can't

Reply:
The solution might be to just add the static cost of hashing a block header and an extra storage read to the open_try and open_ack generated weights.
Those are the significant operations that happen when using the host_consensus_state_proof.

Reply:
Yeah would be nice to have a proxy there

Reply:
I I'll write a separate benchmark for the `host_consensus_state` function, I could access frame system, if I use the raw storage API.